<launch>
    <rosparam command="load" file="$(find car)/config/end_to_end_lane_follow_settings.yaml"/>
    <node pkg="car" name="end_to_end_lane_follow" type="end_to_end_lane_follow" output="screen">
    </node>
</launch>
